README.md

```diff
@@ -19,6 +19,12 @@
 
 gfx-rs is a low-level, cross-platform graphics and compute abstraction library in Rust. It consists of the following components:
 
+## gfx-hal deprecation
+
+As of the v0.9 release, gfx-hal is now deprecated and in maintenance mode. It is the process of being replaced with a new hal that lives in the [`wgpu`](https://github.com/gfx-rs/wgpu) repo called `wgpu-hal`. It will be the hal that will power wgpu and gfx-portability, and will be available for direct use. Read more about the transition in https://github.com/gfx-rs/gfx/discussions/3768. 
```

Comment:
Similarly here, we should soften the wording about gfx-portability, or maybe refactor the whole paragraph a bit.
> gfx-hal was mainly driven by [wgpu](...), which has now switched to its own GPU abstraction called [wgpu-hal](..link to the PR..). For this reason, gfx-hal development is switched to maintenance only, until the developers figure out the story for [gfx-portability](..). Read more about the transition in https://github.com/gfx-rs/gfx/discussions/3768.